src/components/common/inputs/Input/Input.tsx

```diff
@@ -14,7 +14,7 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
   isValid: boolean;
   inputLabel?: string;
   errorMessage?: string;
-  isCountValue: boolean;
+  isCountValue?: boolean;
```

Comment:
```suggestion
  isCountValue: boolean;
```

이거 ? 빼고 false주면 글자수 체크 없어지긴 합니다!